src/data/types.js

```diff
@@ -41,7 +41,8 @@
  * @property {string} [commands[].extraInstruction] human-readable additional instruction to follow
  * @property {object[]} assertions[]
  * @property {1 | 2} assertions[].priority
- * @property {string} assertions[].expectation
+ * @property {string} [assertions[].expectation] - Expected result, required if assertionStatement is not provided.
+ * @property {string} [assertions[].assertionStatement] - Assertion statement, required if expectation is not provided.
```

Comment:
This is not reflected in [the canonical definition of the type](https://github.com/w3c/aria-at/blob/ab77d47ab19db71c635c9bb459ba5c34182e1400/types/aria-at-file.js), contradicting the single-line comment which precedes this multi-line comment. @howard-e can you see about updating ARIA-AT as necessary? Does this need to be reflected in [the Test Format V2 design document](https://github.com/w3c/aria-at/wiki/Test-Format-Definition-V2)?

Reply:
> contradicting the single-line comment which precedes this multi-line comment

`expectation` is a derived value that came from the assertions found in a [v1 tests.csv files format](https://github.com/w3c/aria-at/wiki/Test-Format-V1-Definition#assertion1-through-assertion6). `expectation` is synonymous with `assertionStatement` in v2 and could _technically_ still be referenced through the `expectation` value but keeping them separate is preferred (to me) since it will line up with a known *.csv property for easier referencing. The comment indicates that this is the case, kinda. But it also makes it seem like there could be a case where `expectation` and `assertionStatement` could exist simultaneously for a version of a single test plan.

> @howard-e can you see about updating ARIA-AT as necessary?

Can do!

> Does this need to be reflected in [the Test Format V2 design document](https://github.com/w3c/aria-at/wiki/Test-Format-Definition-V2)?

I think my above comment might not require this but just to be clear, on reflecting [assertionStatement](https://github.com/w3c/aria-at/wiki/Test-Format-Definition-V2#assertionstatement)?

Reply:
> I think my above comment might not require this but just to be clear, on reflecting [assertionStatement](https://github.com/w3c/aria-at/wiki/Test-Format-Definition-V2#assertionstatement)?

Yup, thanks!

Reply:
I attempted to clarify the comments to make it clear when to expect which and that they exist exclusive of one another. Let me know if there is any other change needed in this repo.

Reply:
Follow-up: https://github.com/w3c/aria-at/pull/1025